lib/aruba/processes/spawn_process.rb

```diff
@@ -83,7 +83,7 @@ def start
             sleep startup_wait_time
           end
         rescue ChildProcess::LaunchError => e
-          raise LaunchError, "It tried to start #{cmd}. " + e.message
+          raise LaunchError, "It tried to start #{commandline}. " + e.message
```

Comment:
The variable `cmd` doesn't exist. The exception happens on AppVeyor, so the error is triggered there.